exporter/exporterhelper/consumers_queue.go

```diff
@@ -14,10 +14,10 @@
 
 package exporterhelper
 
-// consumersQueue is largely based on queue.BoundedQueue and matches the subset used in the collector
+// ConsumersQueue is largely based on queue.BoundedQueue and matches the subset used in the collector
 // It describes a producer-consumer exchange which can be backed by e.g. the memory-based ring buffer queue
 // (queue.BoundedQueue) or via disk-based queue (persistentQueue)
-type consumersQueue interface {
+type ConsumersQueue interface {
```

Comment:
Does this need to be exporter? I cannot see it referenced anywhere.

Reply:
That's true, I overlooked it. I will remove it